content/en/docs/setup/release/notes.md

```diff
@@ -259,7 +225,7 @@ filename | sha256 hash
 
 # v1.12.0-beta.2
 
-[Documentation](https://docs.k8s.io) & [Examples](https://releases.k8s.io/release-1.12/examples)
+
 
```

Comment:
Revert all fixed in this PR please, all fixed in https://github.com/kubernetes/website/pull/10692. Could you like to help fix the `Example` link?

Reply:
@stewart-yu @MengZn where should examples link to? Should we just remove it?

Reply:
@jimangel Is it better to reconnect Example to this https://github.com/kubernetes/website/tree/release-1.12/content/en/examples?

Reply:
I'm not sure, considering that directory says "These tests are importing code from kubernetes that isn't really meant to be used outside the repo." and the original link is to (https://releases.k8s.io which is the k/k github release page). It almost looks like the "examples" don't exist anymore. Thoughts @stewart-yu ?

Reply:
@jimangel I have confirmed that there are no examples some versions of [ kubernetes repo](https://github.com/kubernetes/kubernetes), I suggest you can use this [kubernetes website](https://github.com/kubernetes/website/tree/release-1.12/content/en/examples) ?